content/community/changelog/app-nuget/v4/whats-new/_index.en.md

```diff
@@ -4,6 +4,10 @@ description: Overview of changes introduced in version 4.
 toc: true
 ---
 
+## 4.32.0 (14.03.2022) - Person lookup service client
+The platform application Register has been given a new endpoint that can be used to verify correct national identity number. This version of the app template packages have a new person lookup client that can be used to perform lookups with the new endpoint in Register. (This was first implemented in 4.29.0, but bugs pushed it back to 4.32.0.)
```

Comment:
Usikker på hvor mye verdi siste linjen gir, gitt at 4.29.0 ikke er dokumentert her i changelogen og det aldri ble pushet final nugets for den?

Reply:
It was primarily to explain the alpha packages, but I've probably missed the mark. I'll remove it again.